muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/organization/body/application/courses.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import {StateType} from '~/reducers';
+import {connect, Dispatch} from 'react-redux';
+import ApplicationList, { ApplicationListItem } from '~/components/general/application-list';
+import Course from './courses/course';
+
+
+
+interface CoursesProps {
+  
+}
+
+interface CoursesState {
+}
+
+const courses = [
+  {
+    name: "course",
+    description: "Morbi lacus tellus, scelerisque in lacus nec, ullamcorper suscipit velit. Curabitur arcu quam, laoreet sagittis posuere sit amet, dapibus a arcu. Integer tempor orci eu est feugiat, a mollis est aliquet. Donec fringilla sem a risus maximus volutpat",
+  },
+  {
+    name: "course",
+    description: "Morbi lacus tellus, scelerisque in lacus nec, ullamcorper suscipit velit. Curabitur arcu quam, laoreet sagittis posuere sit amet, dapibus a arcu. Integer tempor orci eu est feugiat, a mollis est aliquet. Donec fringilla sem a risus maximus volutpat",
+  },
+  {
+    name: "course",
+    description: "Morbi lacus tellus, scelerisque in lacus nec, ullamcorper suscipit velit. Curabitur arcu quam, laoreet sagittis posuere sit amet, dapibus a arcu. Integer tempor orci eu est feugiat, a mollis est aliquet. Donec fringilla sem a risus maximus volutpat",
+  },
+  ]
+
+class Courses extends React.Component<CoursesProps, CoursesState> {
+  
+  render(){
+    return (
+        <ApplicationList>
+          {courses.map((course:any, index)=>{
```

Comment:
Please don't use the "any" key here, the usage of any should be avoided unless you are casting types or typescript becomes stubborn regarding defined data.

Reply:
I realized just now that I need to remove the mock data.